packages/webpack/src/utils/config.ts

```diff
@@ -57,31 +57,6 @@ export function fileName (ctx: WebpackConfigContext, key: string) {
 }
 
 export function getWebpackConfig (ctx: WebpackConfigContext): Configuration {
-  const { options, config } = ctx
-
-  // TODO
-  const builder = {}
-  const loaders: any[] = []
-
-  // @ts-expect-error TODO: remove support for `build.extend` in v3.5
-  const { extend } = options.build
```

Comment:
Shall we consider making a warning placeholder at least? This will be a silent deprecation with unnoticed regressions if a legacy module exists in projects...

Reply:
good idea. do you think it's worth going further and keeping the behaviour for another minor?

Reply:
Up to you. I am pretty sure there are used modules depending on this and big projects depending on them. Warn + wait for another minor would be good idea too :)